assets/prototype/domain/eventEditor/datetimes/DatesList/AddNewDateButton.tsx

```diff
@@ -17,18 +17,14 @@ const btnRowStyle = {
 	width: '100%',
 };
 
-const AddNewDateButton = () => {
+const AddNewDateButton: React.FunctionComponent = () => {
 	const editorState: EditorState = useEditorModalState();
 	const editorId: EditorId = useDateEditorId('add-new-date', 'xyz');
 
-	const onClick = (e) => {
-		e.preventDefault();
-		e.stopPropagation();
+	const onClick = (): void => {
 		editorState.openEditor(editorId);
 	};
-	const onClose = (e) => {
-		e.preventDefault();
-		e.stopPropagation();
+	const onClose = (): void => {
```

Comment:
@tn3rb if this change makes sense, then I will need to go and apply this on `AddNewTicketButton`

Reply:
done... pushed change